src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibratorModelOutputUnitTest.java

```diff
@@ -24,7 +24,7 @@
     private final double dirichlet = 0.001;
     private final double priorCounts = 20.0;
     private final double epsilon = 1e-6;
-    private final String modelReportName = "vqsr_model.report";
+    private final File modelReportFile = createTempFile("vqsr_model", "report");
```

Comment:
This is actually shared between two tests, but it requires that it is the same file (actually, it was using the same file before too). The test `testVQSRModelOutput` is writing the report, which is used in the `testVQSRModelInput` to test its content (it has also the `dependsOnMethods = {"testVQSRModelOutput"}` property, because it needs this file).

I didn't take any action, because using a different file will break the test (I thought the same as you before and the test failed when I try, until I realized about the dependence).

Reply:
Right, fair point. It looks like it is indeed asserting that it happens first.